### PR TITLE
Blog post for 0.6.0 release with python sdk

### PR DESCRIPTION
**Pull request by aaltay**
R: @davorbonaci

### CODE REVIEW

File: src/_data/authors.yml [hunk 1]
```diff
@@ -40,3 +40,6 @@ klk:
     name: Kenneth Knowles
     email: klk@apache.org
     twitter: KennKnowles
+altay:
```
Comment (davorbonaci) [inline src/_data/authors.yml]: Alphabetical?

File: src/_posts/2017-03-16-python-sdk-release.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - altay
+---
+
+Apache Beam’s latest release, version [0.6.0](https://beam.apache.org/get-started/downloads/), introduces a new SDK -- this time, for the Python programming language. The Python SDK joins the Java SDK as the second implementation of the Beam programming model.
+
+<!--more-->
+
+The Python SDK incorporates all of the main concepts of the Beam model, including ParDo, GroupByKey, Windowing, and others. It features extensible IO APIs for writing bounded sources and sinks, and provides built-in implementation for reading and writing Text, Avro, and TensorFlow record files, as well as connectors to Google BigQuery and Google Cloud Datastore.
+
+There are two runners capable of executing pipelines written with the Python SDK today: [Direct Runner](https://beam.apache.org/documentation/runners/direct/) and [Dataflow Runner](https://beam.apache.org/documentation/runners/dataflow/). These runners are currently limited to batch only execution. Upcoming features will shortly bring the benefits of the Python SDK to additional runners.
```
Comment (davorbonaci) [inline src/_posts/2017-03-16-python-sdk-release.md]: ... and Dataflow Runner, both of which are currently limited to batch execution only.

File: src/_posts/2017-03-16-python-sdk-release.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Try the Apache Beam Python SDK
+
+If you would like to try out the Python SDK, a good place to start is the [Quickstart](https://beam.apache.org/get-started/quickstart-py/). After that, you can take a look at additional [examples](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples), and deep dive into the [API reference](https://beam.apache.org/documentation/sdks/pydoc/).
```
Comment (davorbonaci) [inline src/_posts/2017-03-16-python-sdk-release.md]: pin to 0.6.0 version, not master

**Comment by aaltay** [comment]
Thank you @davorbonaci, addressed your comments. PTAL.

**Pull request closed without merging**